src/puppetlabs/puppetdb/anonymizer.clj

```diff
@@ -31,11 +32,10 @@
    (contains? event "old_value")
    (contains? event "resource_type")))
 
-(defn report?
+(pls/defn-validated report? :- report/report-schema
```

Comment:
Nevermind, keyword is right yeah ... sorry. Umm. This still feels wrong, hang on let me think about this.